documentation/examples/rbac-setup.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+  namespace: default
```

Comment:
In the [Role Binding Examples](https://kubernetes.io/docs/admin/authorization/rbac/#role-binding-examples) section it has an example of how to bind a ServiceAccount and it specifies the namespace. 

Service accounts are always tied to a namespace, whereas the ClusterRoleBinding itself isn't. You can reuse the same service account name in multiple namespaces. So I guess it's mandatory to specify which namespace to use for a service account in the ClusterRoleBinding. I'll test without to see what happens.